resources/assets/less/bem/multiplayer-room.less

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.multiplayer-room {
+  --bg: hsla(var(--hsl-b6), 0.75);
+  --border-radius: @border-radius--large;
+
+  display: flex;
+  font-size: @font-size--normal;
+
+  position: relative;
+
+  color: hsl(var(--hsl-c1));
+
+  &__chat-button {
+    .reset-input();
+  }
+
+  &__content {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background: linear-gradient(0.25turn, hsl(var(--hsl-b2)), hsla(var(--hsl-b2), 0.8));
+    border-radius: var(--border-radius);
+
+    padding: 6px 20px;
+    margin-left: 20px;
+    width: calc(100% - $margin-left);
+  }
+
+  &__cover {
+    .full-size();
+    object-fit: cover;
+    border-radius: var(--border-radius);
+
+    &--default {
+      .at2x('~@images/layout/beatmaps/default-bg.png');
+      background-position: center;
+      background-size: cover;
```

Comment:
Should be repeating pattern and not oversized icons...

Reply:
so apparently it's fine in card because the max size in there is about the same as the image itself :upside_down_face: while in here it goes all the way to 700+px